### DISCUSSION TITLE
Discord Embeds list Job name as "None"

### DISCUSSION THREAD

**Discussion by cm8263** [community/community#41460] [Actions] [answered]
### Select Topic Area

Bug

### Body

Some time between December 7th and December 10th the job names of my actions started showing as "None" in my Discord Embeds. The Embeds are set up as a Webhook on the repository in question. See the images below.

![image](https://user-images.githubusercontent.com/10535902/206940851-d9aca1f4-2963-4f6b-95c7-3c01c4eceb92.png)

I have not made any recent changes to the [workflow file](https://github.com/cm8263/Purpose/blob/feature/dialog/.github/workflows/butler.yml).
The actions can be found [here](https://github.com/cm8263/Purpose/actions).

Cheers,
Christopher M.

**Comment by Aha00a** [comment]
![image](https://user-images.githubusercontent.com/1396800/206981538-ea3cdef4-7091-4944-b1db-03ea2e97bd8e.png)

**Comment by anthonypillot** [comment]
Hello. Same problem at exactly the same time. No changes have been made to the workflow.

**Comment by jfabellera** [comment]
Same here

**Comment by digidigo** [comment]
Same here - I would think that this could be on the Discord side?

**Reply by cm8263** [reply]
> Same here - I would think that this could be on the Discord side?

I contacted Discord support on the 14th and at this stage I'm still waiting for a reply from their Development team.
I will relay any information they provide ([other than telling me to restart my PC and disable my anti-virus to see if that "fixes" it -_-](https://i.imgur.com/1EW7iDW.png)).

**Comment by cm8263** [comment, marked as answer]
The issue appears to have been resolved. No word from Discord support at this stage.

**Reply by jfabellera** [reply]
Looks like it was on Discord's end:
https://github.com/discord/discord-api-docs/issues/5763
